starwars/src/components/CharacterList.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'; 
+import Character from './Character';
+
+const CharacterList = props => {
+    return props.list.map(character => { 
+        return (
+            <div className ={`characterCard  ${character.name.replace(" ", "")}`}>
+            <Character 
+                header = {character.name}
+                birthYear = {character.birth_year}
+                gender = {character.gender}
+                eyeColor = {character.eye_color}
+                hair = {character.hair_color}
+                height = {character.height}
+                mass = {character.mass}
+                />
```

Comment:
Rather than passing each individual key of your object, you can just pass the whole object and access the key:value pairs from inside the child component.